### PR TITLE
Improve the gallery example for TypeDecorator

### PR DESCRIPTION
**Pull request by adrien-berchet**
Following #371 to improve the example. WDYT @tiptop96?

### CODE REVIEW

File: tests/gallery/test_type_decorator.py [hunk 1]
```diff
@@ -31,33 +31,53 @@ class TransformedGeometry(TypeDecorator):
...
         self.app_srid = app_srid
         self.db_srid = db_srid
 
     def column_expression(self, col):
-        """The column_expression() method is overridden to ensure that the
-        SRID of the resulting WKBElement is correct"""
+        """The column_expression() method is overridden to set the correct type.
+
+        This is needed so that the returned element will also be decorated. In this case we don't
+        want to transfor is again afterwards so we set the same SRID to both the ``db_srid`` and
```
Comment (tiptop96) [inline tests/gallery/test_type_decorator.py:46]: Meant to say "transform it"?

**Comment by tiptop96** [comment]
One comment about a possible spelling error, LGTM! 👍

File: tests/gallery/test_type_decorator.py (hunk 1 above)
Comment (adrien-berchet) [reply tests/gallery/test_type_decorator.py]: Indeed, thanks!

**Comment by adrien-berchet** [comment]
> One comment about a possible spelling error, LGTM! +1

Perfect, thanks :-)

**Pull request merged**